frontstage/templates/secure-messages-create.html

```diff
@@ -15,28 +19,45 @@ <h1 class="saturn">Create message</h1>
                 <form method="post">
                     <p>To: ONS Business Surveys team</p>
 
+                    {% if "subject" in errors %}
+                    <div class="panel panel--simple panel--error panel--spacious">
+                    <p class="error-message">{{ errors['subject'] }}</p>
+                    {% endif %}
+
                     <fieldset>
                         <label for="secure-message-subject">Subject</label>
                         <input
                             id="secure-message-subject"
                             class="secure-message-form__subject input input--text"
                             type="text"
                             name="secure-message-subject"
-                            required
-                            maxlength="100"/>
+                            value="{{draft['subject']}}" />
                     </fieldset>
 
+                    {% if "subject" in errors %}
+                    </div>
+                    {% endif %}
+
                     <br />
 
+                    {% if "body" in errors %}
+                    <div class="panel panel--simple panel--error panel--spacious">
+                        <p class="error-message">{{ errors['body'] }}</p>
+                    {% endif %}
+
                     <fieldset>
                         <label for="secure-message-body">Message</label>
                         <textarea
                             id="secure-message-body"
                             class="secure-message-form__body input"
                             rows="10"
-                            name="secure-message-body"
-                            maxlength="10000"></textarea>
+                            name="secure-message-body">{{ draft ['body'] }}</textarea>
                     </fieldset>
+
+                    {% if "body" in errors %}
+                    </div>
+                    {% endif %}
```

Comment:
creates a closing div for the error box. It's not very elegant but works for now, the other frontstage form pages do a similar thing